api/client.js

```diff
@@ -204,54 +209,58 @@ class Client {
       can check if that's the case using client.isRedirectionError(err). If it
       is a redirection error, information about available servers is stored in
       err.servers and err.caCert (if a certificate is required).
+    @return {Promise} This promise will be rejected if there is an error
+      connecting, or resolved with a new connection instance.
```

Comment:
Please mention that the promise will not be resolved or rejected if a callback is provided.